src/types/catalog.d.ts

```diff
@@ -47,6 +47,12 @@ export interface ShopperCatalogReleaseBase extends Identifiable {
       }
     }
   }
+  meta: {
+    created_at: string
+    is_full_publish: boolean
+    release_status: string
```

Comment:
Do we have a defined list of what these `relase_status` can be? That way we can have a type declaration something like:

```
export type CatalogReleaseStatus = 'PUBLISHED' | 'SOME_STATUS' | 'SOME_OTHER_STATUS'
```

```
meta {
 ...
 release_status: CatalogReleaseStatus
 ...
}
```

This would allow the user of the sdk not to have to look up or guess what the status can or can't be and get a better IDE experience.

My preference would be have more constrained types where possible. What do you think?